lib/rules/lines-around-comment.js

```diff
@@ -115,6 +125,11 @@ module.exports = function(context) {
             after = false;
         }
 
+        // if block start is allowed and comment is at block start then return
+        if (options.allowBlockStart && isCommentAtBlockStart()) {
```

Comment:
Won't exiting early fail to catch a missing line after a comment in an otherwise empty block?

```js
// "lines-around-comment": [2, {
//     "beforeLineComment": true,
//     "afterLineComment": true,
//     "allowBlockStart": true
// }]
while (true) {
    // This should be followed by a blank line
}
```